ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/error/PostSSOErrorTest.kt

```diff
@@ -63,18 +58,18 @@ class PostLoginTest : StringSpec() {
         private fun createValidAuthnRequest(): String {
             val authnRequest = AuthnRequestBuilder().buildObject().apply {
                 issuer = IssuerBuilder().buildObject().apply {
-                    value = TestCommon.SP_ISSUER
+                    value = SP_ISSUER
                 }
-                assertionConsumerServiceURL = acsUrl[HTTP_POST]
-                id = TestCommon.ID
+                assertionConsumerServiceURL = acsUrl[SamlProtocol.Binding.HTTP_POST]
+                id = ID
                 version = SAMLVersion.VERSION_20
                 issueInstant = DateTime()
-                destination = Common.getSingleSignOnLocation(POST_BINDING)
-                protocolBinding = POST_BINDING
+                destination = Common.getSingleSignOnLocation(SamlProtocol.POST_BINDING)
+                protocolBinding = SamlProtocol.POST_BINDING
```

Comment:
✏️ You could import `org.codice.security.saml.SamlProtocol.POST_BINDING` and `org.codice.security.saml.SamlProtocol.Binding.HTTP_POST` to make it cleaner in the test like we have it in the positive tests